src/main/java/com/team1816/season/Robot.java

```diff
@@ -596,7 +621,7 @@ public void manualControl() {
         }
 
         drive.setTeleopInputs(
-            controlBoard.getAsDouble("throttle"),
+            -controlBoard.getAsDouble("throttle"),
```

Comment:
This is ok for now but ultimately; we should not have to do this.  We know if motors are inverted etc that should drive the need for this

Reply:
Y axis in general should be negated, input from controller is opposite of logical. This is why the negation is applied right now. Motor inversion is fine, positive input to drive results in forward motion.